http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -82,9 +73,13 @@
 
     // Override connection idle timeout for Netty http client to reduce the frequency of "server failed to complete the
     // response error". see https://github.com/aws/aws-sdk-java-v2/issues/1122
+    // CHECKSTYLE:OFF - Lines are too long
     private static final AttributeMap NETTY_HTTP_DEFAULTS = AttributeMap.builder()
-                                                                        .put(CONNECTION_MAX_IDLE_TIMEOUT, Duration.ofSeconds(5))
+                                                                        .put(
+                                                                            SdkHttpConfigurationOption.CONNECTION_MAX_IDLE_TIMEOUT,
```

Comment:
This formatting is weird

Reply:
Yaaa, normal format puts it past the line limit. Disable Checkstyle for this block?

Reply:
New line AttributeMap.builder()?

Reply:
Works for me!